lib/board.rb

```diff
@@ -1,11 +1,23 @@
 class Board
-  attr_reader :board
+  attr_reader :board, :moves_remaining
 
   def initialize(board)
```

Comment:
wondering if this could be squares, as it feels odd to have a Board class that takes a "board" as an argument.

Reply:
Good point. I'll change this to squares. Thanks, Matt